### DISCUSSION TITLE
Discussion on Opensource with Daily Dev

### DISCUSSION THREAD

**Discussion by anushkrishnav** [EddieHubCommunity/support#787] [Show and tell]
Having a Live discussion on Opensource with Ido Shamun Co-founder and  CTO of [daily dev](http://daily.dev/), 
🗓 Date: 27th January, 2020
⏰ Time: 3:00 PM IST
Host: @anushkrishnav 
🔗 Register yourself for the session here: https://forms.gle/dD5KvE3idvcfW3UFA
🖇 Set your reminders here: https://youtu.be/okBDedj7Iz0
For more details: https://twitter.com/Anush_krishna_v/status/1354152528674340864?s=20

**Comment by BOLT04** [comment]
Great video @anushkrishnav 👍. I didn't know much about daily dev, and the fact that it's open-source is even cooler 😃.
Thanks for sharing 👍!

**Reply by anushkrishnav** [reply]
Thank you so much BOLT will be having talks with more cool opensource org do recommend a project or company !!

**Comment by eddiejaoude** [comment]
Oh I missed this, I will have to watch it and catch up 👍

**Reply by anushkrishnav** [reply]
Awesome would love to know your comments on it !!
